src/Altinn.App.Core/Models/Layout/Components/BaseComponent.cs

```diff
@@ -19,15 +19,15 @@ public class BaseComponent
     /// <summary>
     /// Constructor that ensures n
```

Comment:
Complete comment:)

Reply:
This comment is on an old diff. Currently the comment is 
```csharp
/// <summary>
/// Constructor
/// </ summary>
```
Not sure how to really add value, but i changed it to `Constructor for <see cref="BaseComponent" />`?